tools/test_image.py

```diff
@@ -49,12 +50,21 @@
     print(f'   Exception: {err}')
     destroy_droplet_and_exit(droplet)
 
+if droplet.load().ip_address is None:
+    print('Waiting for ip address from droplet')
+    IP = wait_for_droplet_ip(droplet, timeout_seconds=600)
+    if IP == STATUS_OK:
+        print(f'   Droplet IP: {droplet.ip_address}')
+    else:
+        print('   Timeout waiting for IP check')
+        destroy_droplet_and_exit(droplet)
+
```

Comment:
The if is not needed. `IP` naming is misleading to what it actually returns. maybe `droplet_status` ?

```suggestion
    print('Waiting for ip address from droplet')
    IP = wait_for_droplet_ip(droplet, timeout_seconds=600)
    if IP == STATUS_OK:
        print(f'   Droplet IP: {droplet.ip_address}')
    else:
        print('   Timeout waiting for IP check')
        destroy_droplet_and_exit(droplet)
```

Reply:
If the IP is already available no need to call `wait_for_droplet_ip` that why the if is for

Reply:
Since it is not common that the IP address is already found and that the if condition is present in the function anyway we avoid duplicating code and add complexity